docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -279,6 +295,10 @@ public CreateContainerCmd fillContainerConfig(CreateContainerCmd containerConfig
         if (environment != null && environment.length > 0)
             containerConfig.withEnv(environment);
 
+        if (getMacAddress() != null && !getMacAddress().trim().isEmpty()) {
```

Comment:
I don't like guava functions internals. Jenkins has Utils.* And all input variables handling need to be reviewed later. So this not critical atm.